debug.log

```diff
@@ -0,0 +1 @@
+[0223/161757.992:ERROR:registration_protocol_win.cc(102)] CreateFile: The system cannot find the file specified. (0x2)
```

Comment:
looks like this should be excluding. but i'm guessing its already in Master

Reply:
I don't believe I generated this myself so I think it's already there. Yeah it is.

On a subsequent submit, I can remove it. 

Thanks.